### ISSUE TITLE
Typescript `no-floating-promises`  causes loading rule error

### ISSUE THREAD

**Issue by hmajid2301** [sveltejs/eslint-plugin-svelte#435]
### Before You File a Bug Report Please Confirm You Have Done The Following...

- [X] I have tried restarting my IDE and the issue persists.
- [X] I have updated to the latest version of the packages.

### What version of ESLint are you using?

8.33.0

### What version of `eslint-plugin-svelte` are you using?

2.15.0

### What did you do?

<details>
<summary>Configuration</summary>

```
<!-- Paste your configuration here -->
module.exports = {
	root: true,
	parser: "@typescript-eslint/parser",
	parserOptions: {
		project: "./tsconfig.json",
		extraFileExtensions: [".svelte"]
	},
	extends: [
		"eslint:recommended",
		"plugin:svelte/recommended",
		"plugin:@typescript-eslint/recommended",
		"plugin:@typescript-eslint/recommended-requiring-type-checking",
		"prettier",
		"plugin:import/errors",
		"plugin:import/warnings",
		"plugin:import/typescript"
	],
	plugins: ["@typescript-eslint"],
	ignorePatterns: ["*.cjs", "svelte-french-toast"],
	overrides: [
		{
			files: ["*.svelte"],
			parser: "svelte-eslint-parser",
			parserOptions: {
				parser: "@typescript-eslint/parser"
			}
		}
	],
	rules: {
		"import/no-unresolved": "off",
		"import/order": [
			"warn",
			{
				groups: ["builtin", "external", ["sibling", "parent"], "index"],
				pathGroups: [
					{
						pattern: "$app/**",
						group: "external"
					},
					{
						pattern: "~/**",
						group: "sibling"
					}
				],
				alphabetize: {
					order: "asc",
					caseInsensitive: true
				},
				"newlines-between": "always"
			}
		]
	},
	settings: {
		"import/ignore": ["svelte-french-toast"],
		"svelte3/typescript": () => require("typescript")
	},
	parserOptions: {
		sourceType: "module",
		ecmaVersion: 2020
	},
	env: {
		browser: true,
		es2017: true,
		node: true
	}
};

```
</details>

```svelte

```

### What did you expect to happen?

For the linter to able to run to look for incorrect promises

### What actually happened?

The following error was thrown:

```bash
pnpm run lint

> bookmarkey-gui@0.0.1 lint /home/haseeb/projects/bookmarkey/gui
> eslint .

=============

WARNING: You are currently running a version of TypeScript which is not officially supported by @typescript-eslint/typescript-estree.

You may find that it works just fine, or you may not.

SUPPORTED TYPESCRIPT VERSIONS: >=3.3.1 <5.0.0

YOUR TYPESCRIPT VERSION: 5.0.2

Please only submit bug reports when using the officially supported version.

=============

Oops! Something went wrong! :(

ESLint: 8.33.0

Error: Error while loading rule '@typescript-eslint/await-thenable': You have used a rule which requires parserServices to be generated. You must therefore provide a value for the "parserOptions.project" property for @typescript-eslint/parser.
Occurred while linting /home/haseeb/projects/bookmarkey/gui/playwright.config.ts
    at Object.getParserServices (/home/haseeb/projects/bookmarkey/gui/node_modules/.pnpm/@typescript-eslint+utils@5.50.0_eslint@8.33.0_typescript@5.0.2/node_modules/@typescript-eslint/utils/dist/eslint-utils/getParserServices.js:22:15)
    at create (/home/haseeb/projects/bookmarkey/gui/node_modules/.pnpm/@typescript-eslint+eslint-plugin@5.50.0_@typescript-eslint+parser@5.50.0_eslint@8.33.0_typescript@5.0.2/node_modules/@typescript-eslint/eslint-plugin/dist/rules/await-thenable.js:44:37)
    at Object.create (/home/haseeb/projects/bookmarkey/gui/node_modules/.pnpm/@typescript-eslint+utils@5.50.0_eslint@8.33.0_typescript@5.0.2/node_modules/@typescript-eslint/utils/dist/eslint-utils/RuleCreator.js:41:20)
    at createRuleListeners (/home/haseeb/projects/bookmarkey/gui/node_modules/.pnpm/eslint@8.33.0/node_modules/eslint/lib/linter/linter.js:926:21)
    at /home/haseeb/projects/bookmarkey/gui/node_modules/.pnpm/eslint@8.33.0/node_modules/eslint/lib/linter/linter.js:1108:110
    at Array.forEach (<anonymous>)
    at runRules (/home/haseeb/projects/bookmarkey/gui/node_modules/.pnpm/eslint@8.33.0/node_modules/eslint/lib/linter/linter.js:1045:34)
    at Linter._verifyWithoutProcessors (/home/haseeb/projects/bookmarkey/gui/node_modules/.pnpm/eslint@8.33.0/node_modules/eslint/lib/linter/linter.js:1397:31)
    at Linter._verifyWithConfigArray (/home/haseeb/projects/bookmarkey/gui/node_modules/.pnpm/eslint@8.33.0/node_modules/eslint/lib/linter/linter.js:1772:21)
    at Linter.verify (/home/haseeb/projects/bookmarkey/gui/node_modules/.pnpm/eslint@8.33.0/node_modules/eslint/lib/linter/linter.js:1479:65)
 ELIFECYCLE  Command failed with exit code 2.
task: Failed to run task "lint": exit status 1
````

### Link to **GitHub Repo** with Minimal Reproducible Example

https://gitlab.com/bookmarkey/gui/-/merge_requests/156

git clone https://gitlab.com/bookmarkey/gui.git
git checkout origin/feat/#213
pnpm install
pnpm run lint

**Comment by ota-meshi** [comment]
I think you need to remove duplicate parserOptions or merge them.
